profiles/digitalocean/centos_7.go

```diff
@@ -142,5 +145,15 @@ func NewCentosCluster(name string) *cluster.Cluster {
 	c := cluster.NewCluster(name)
 	c.SetProviderConfig(controlPlaneProviderConfig)
 	c.NewMachineSetsFromProviderConfigs(machineSetsProviderConfigs)
+
+	secret := &apiv1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "digitalocean",
+			Namespace: "kube-system",
+		},
+		Data: map[string][]byte{"access-token": []byte(os.Getenv("DIGITALOCEAN_ACCESS_TOKEN"))},
```

Comment:
I think you can solve the double-base64 encoding by using `StringData` instead of `Data` here.